keen/explorer
# Fixes #142

## What does this PR do?
This fixes #142 Funnel step filters not properly parsed from URL.

## How can I test this?
1) Create a funnel query and have one of the steps have a List filter.
2) Run the query. Notice it works as expected.
3) Refresh the page.
4) Open the filters panel for that filter step and notice the List filter is properly setup.
